Marlin/src/gcode/feature/pause/M600.cpp

```diff
@@ -58,8 +62,21 @@
 void GcodeSuite::M600() {
   point_t park_point = NOZZLE_PARK_POINT;
 
-  const int8_t target_extruder = get_target_extruder_from_command();
-  if (target_extruder < 0) return;
+  #if ENABLED(MIXING_EXTRUDER)
+    const int8_t target_e_stepper = get_target_e_stepper_from_command();
+    if (target_e_stepper < 0) return;
+
+    const uint8_t old_mixing_tool = mixer.get_current_vtool();
+    mixer.T(MIXER_DIRECT_SET_TOOL);
```

Comment:
This update to the tool index is optional since the virtual tool index isn't needed by the `mixer` methods used here that set up the current integer mix proportions. The `mixer.T` below is still needed to get back to the old normalized mix. I'll leave it in for future-proofing for now.

Reply:
I don't follow. If we leave the tool index alone, the call to `mixer.normalize()` below will modify the mix proportions for the current virtual tool, making impossible to get back back to the proportions that were in effect prior to calling `M600`. Unless I'm misunderstanding how `mixer.normalize()` works, we need to save the tool index, change it to something else, and then set it back when we're done. The whole point of the `MIXER_DIRECT_SET_TOOL` tool is to give us a sandbox where we can change the mix to whatever is needed for actions like `M600`, `M701`, and `M702` without messing with the user's tools.

Reply:
Unless I'm mistaken, the tool index simply isn't used by any of the methods called here. When is `normalize` called by `M600`?

Reply:
It's been a few days out of the loop. I'll do one more review and see where things are at.